package.json

```diff
@@ -0,0 +1,5 @@
+{
+  "dependencies": {
+    "sw-precache": "^5.2.1"
```

Comment:
I see you removed the package.lock 🤔 any particular reason ? Don't we want to lock the dependencies at a specif version until explicitly updated ? 

Do you mind also adding https://github.com/Rebilly/ReDoc/blob/master/cli/README.md to the package.json as well ? This tool is used to check to run then `serve` command to test/check the everything still works :D. Guess I forgot to create a package.json 🤦‍♂️